Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java

```diff
@@ -65,14 +66,13 @@ static String convertStreamToString(InputStream is) {
   }
 
   void authenticationAndBringPages() {
-    Dialog authDialog = new Dialog(mainContext);
+    authDialog = new Dialog(mainContext);
```

Comment:
In this case, `authDialog` is not accessed outside `authenticationAndBringPages`. With the current situation, I think it's enough to have `final` at here.